jwt_c.opam

```diff
@@ -1,15 +1,17 @@
 opam-version: "2.0"
```

Comment:
You could consider switching that to dune 1.10/1.11+ then you can write all that into `dune-project`. But this is purely a suggestion.

Reply:
Should be done now.